src/oic/oauth2/__init__.py

```diff
@@ -839,6 +839,7 @@ def do_access_token_request(
             http_args = ht_args
         else:
             http_args.update(ht_args)
+            http_args.pop("password", None)
```

Comment:
why pop password here?

Reply:
For some reason it was passing through and ended up as an invalid kwarg to requests library...

Reply:
ok. It is wrong in any case, so the fix should do no harm, but probably the issue is burried somewhere in request_info().

Reply:
Yes, my guess as well. But didn't want to spend a huge amount of time on that. Will make a separate issue for that...